tests/rpc/conftest.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import os
+import pytest
+import tempfile
+import uuid
+
+from evm.rpc.ipc import (
+    start,
+)
+
+
+@pytest.yield_fixture(scope='session')
+def event_loop():
+    loop = asyncio.new_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(scope='session')
+def ipc_pipe():
+    tmpdir = tempfile.gettempdir()
+    return os.path.join(tmpdir, 'test-%s.ipc' % uuid.uuid4())
+
+
+@pytest.fixture(scope='session', autouse=True)
+def ipc_server(ipc_pipe, event_loop):
+    server = start(ipc_pipe, loop=event_loop)
+
+    yield
```

Comment:
Seems like you might as well `yield server` here just *because*

Reply:
Is it unreasonable that I want to leave it out because it's not necessary?

I don't want someone (ie~ me in 3 months) to read this and wonder where server is being used. I think if it yields nothing, it's clearer that the point of this fixture is just to start up the server and then shut it down.

Reply:
No, not unreasonable though it might be good to leave a comment indicating this intent.